rust/tvm-rt/src/object/object_ptr.rs

```diff
@@ -147,14 +148,26 @@ impl Object {
     }
 }
 
+// impl fmt::Debug for Object {
```

Comment:
remove dead code

Reply:
I was leaving this here for after I land another outstanding branch so I can clean up some of the code.